backend/tests/v5/e2e/routes/teamspaces/index.test.js

```diff
@@ -131,9 +130,17 @@ const testGetTeamspaceMembers = () => {
 
 				return data;
 			});
-			expectedData.push({ job: DEFAULT_OWNER_JOB, user: teamspace });
 
-			ServiceHelper.outOfOrderArrayEqual(res.body.members, expectedData);
+			const resWithoutAdmin = res.body.members.filter(
```

Comment:
not entirely sure why you had to do this, `outOfOrderArrayEqual` should've done the trick? why do you have to split the array and check it separately?